### PR TITLE
Feature-TG-95: Log in Form

### PR DESCRIPTION
**Pull request by FedeBrus64**
What has been made:
- Form for log in created and styled
- Route for the log in added to routes
- Auth from redux implemented (This still has some errors. I will correct them as soon as possible)
- Validations made using Joi library

### CODE REVIEW

File: src/Components/Employee/Login/index.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { useHistory } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { useForm } from 'react-hook-form';
+import { joiResolver } from '@hookform/resolvers/joi';
+import Input from 'Components/Shared/Input';
+import Button from 'Components/Shared/Button';
+import Modal from 'Components/Shared/Modal/Modal';
+import Loading from 'Components/Shared/Loading';
+import styles from './login.module.css';
+import { login, getUser } from 'redux/auth/thunks';
```
Comment (Julian25) [inline src/Components/Employee/Login/index.jsx:12]: getUser no lo estas usando nunca

**Review by Julian25 — APPROVED** [review]
everything is Ok

File: src/Components/Employee/Login/index.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  const submitHandler = (data) => {
+    dispatch(login(data));
+    setModalMessage(true);
+  };
+
+  const closeHandler = () => {
+    setModalMessage(false);
+    dispatch(resetMessage());
+    if (!error) {
+      goBack();
```
Comment (lucasrendo) [inline src/Components/Employee/Login/index.jsx:62]: This is not where the login should redirect in the case of success, it should redirect to the logged in home

File: src/Components/Employee/Login/index.jsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            </Button>
+            <Button>Log in</Button>
+          </div>
+        </form>
+      )}
+      <Modal handleClose={() => closeHandler()} isOpen={modalMessage} isConfirmation={false}>
+        <h2>{message}</h2>
+      </Modal>
+      <div>
+        Do you not have an account? <a>Sign up</a>
```
Comment (lucasrendo) [inline src/Components/Employee/Login/index.jsx:93]: Este link no va a ningun lado. Deberia ser un Link en vez de un anchor

File: src/Components/Employee/Login/login.module.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+.container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+  
+h2 {
```
Comment (lucasrendo) [inline src/Components/Employee/Login/login.module.css:9]: Los h2 no necesitan ser flex si solo contienen texto adentro

File: src/Components/Tasks/Form/index.jsx [hunk 5]
```diff
@@ -137,7 +137,7 @@ function Tasks() {
             <Button classes={'red'} onClick={() => goBack()}>
               Back
             </Button>
-            <Button>Save</Button>
+            <Button>Log in</Button>
```
Comment (lucasrendo) [inline src/Components/Tasks/Form/index.jsx:140]: Log in en las tasks?

**Review by lucasrendo — COMMENTED** [review]
- El archivo login esta 2 veces, dentro de employee y directamente dentro de components. Login tendria que estar dentro de una carpeta auth en components y tendria que haber 1 solo login

File: src/Components/Employee/Login/login.module.css (hunk 4 above)
Comment (FranGutierrez) [reply src/Components/Employee/Login/login.module.css:9]: Otra cosa, no usen las etiquetas html para darles estilos porque se aplican en toda la app. Usen clases para dar los estilos

**Review by lucasrendo — APPROVED** [review]

**Review by FranGutierrez — APPROVED** [review]

**Pull request merged**